test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -85,6 +86,41 @@ public static bool IsCoreClr()
             return RuntimeFrameworkDescription.Contains("core") || IsNet5();
         }
 
+        public static string GetRuntimeIdentifier()
+        {
+            return IsCoreClr()
+                       ? string.Empty
+                       : $"{EnvironmentTools.GetOS()}-{EnvironmentTools.GetPlatform()}";
+        }
+
+        public static string GetSolutionDirectory()
+        {
+            return EnvironmentTools.GetSolutionDirectory();
+        }
```

Comment:
Didn't I remove this recently so this project didn't need to reference `Datadog.Core.Tools`? or vice-versa?

Reply:
It has been rebased so many times it's possible some changes got lost 🤔  I'll check

Reply:
I wouldn't worry about it tbh, I'm removing that whole project in my nuke-tool-merge branch 😉